doc/release.rst

```diff
@@ -4,6 +4,187 @@ Release Notes
+
+New ``curves`` subpackage
+-------------------------
+
+A new ``curves`` subpackage has been added to preprocess, and extract features from, collections of multi-channel curves such as returned by ``BettiCurve``, ``PersistenceLandscape`` and ``Silhouette`` (`#480 <https://github.com/giotto-ai/giotto-tda/pull/480>`_). It contains:
+
+-  A ``StandardFeatures`` transformer that can extract features channel-wise in a generic way.
+-  A ``Derivative`` transformer that computes channel-wise derivatives of any order by discrete differences (`#492 <https://github.com/giotto-ai/giotto-tda/pull/492>`_).
+
+New ``metaestimators`` subpackage
+---------------------------------
+
+A new ``metaestimator`` subpackage has been added with a ``CollectionTransformer`` meta-estimator which converts any transformer instance into a fit-transformer acting on collections (`#495 <https://github.com/giotto-ai/giotto-tda/pull/495>`_).
+
+Images
+------
+
+-  A ``DensityFiltration`` for collections of binary images has been added (`#473 <https://github.com/giotto-ai/giotto-tda/pull/473>`_).
+-  ``Padder`` and ``Inverter`` have been extended to greyscale images (`#489 <https://github.com/giotto-ai/giotto-tda/pull/489>`_).
+
+Time series
+-----------
+
+-  ``TakensEmbedding`` is now a new transformer acting on collections of time series (`#460 <https://github.com/giotto-ai/giotto-tda/pull/460>`_).
+-  The former ``TakensEmbedding`` acting on a single time series has been renamed to ``SingleTakensEmbedding`` transformer, and the internal logic employed in its ``fit`` for computing optimal hyperparameters is now available via a ``takens_embedding_optimal_parameters`` convenience function (`#460 <https://github.com/giotto-ai/giotto-tda/pull/460>`_).
+-  The ``_slice_windows`` method of ``SlidingWindow`` has been made public and renamed into ``slice_windows`` (`#460 <https://github.com/giotto-ai/giotto-tda/pull/460>`_).
+
+Graphs
+------
+
+-  ``GraphGeodesicDistance`` has been improved as follows (`#422 <https://github.com/giotto-ai/giotto-tda/pull/422>`_):
+
+   -  The new parameters ``directed``, ``unweighted`` and ``method`` have been added.
+   -  The rules on the role of zero entries, infinity entries, and non-stored values have been made clearer.
+   -  Masked arrays are now supported.
+
+-  A ``mode`` parameter has been added to ``KNeighborsGraph``; as in ``scikit-learn``, it can be set to either ``"distance"`` or ``"connectivity"`` (`#478 <https://github.com/giotto-ai/giotto-tda/pull/478>`_).
+
+-  List input is now accepted by all transformers in ``gtda.graphs``, and outputs are consistently either lists or 3D arrays (`#478 <https://github.com/giotto-ai/giotto-tda/pull/478>`_).
+
+-  Sparse matrices returned by ``KNeighborsGraph`` and ``TransitionGraph`` now have int dtype (0-1 adjacency matrices), and are necessarily symmetric (`#478 <https://github.com/giotto-ai/giotto-tda/pull/478>`_).
```

Comment:
Did you mean `(...) are *not* necessarily symmetric`?

Reply:
Great spot! Thanks, will change.